### ISSUE TITLE
Right-click doesn't work in Extended editors (Windows) 

### ISSUE THREAD

**Issue by corvidcase** [standardnotes/forum#857]
In notes using Extended's editors, the right-click context menu doesn't show up. It still works for the basic plain text editor. 

I've tried this against Markdown Pro/Basic/Minimalist/Color, the code editor, and the tasks editor.

I'm fairly certain it isn't the fault of the themes or any of the other extensions. The right-click menu still shows up in the plain text editor even while they're installed, and I also uninstalled them to make sure. When I get back to work on Monday, I'll see if this problem persists on my work computer, just in case it's something wrong with this one. 

Keyboard shortcuts still work, so it doesn't make the app unusable, but it _is_ rather annoying. :( 

Windows 10, Standard Notes version v3.0.25.

**Comment by mobitar** [comment]
Right, right-click menus are not available in editors (which use iframes) due to security restrictions. The only way around this in the future is building a custom right-click menu that each editor implements. Most likely the direction we'll be heading in.
